src/qililab/instruments/rohde_schwarz/sgs100a.py

```diff
@@ -106,6 +116,12 @@ def set_parameter(self, parameter: Parameter, value: ParameterValue, channel_id:
                 else:
                     self.turn_off()
             return
+        if parameter == Parameter.IQ_STATE:
```

Comment:
This enables hardware modulation, right? If yes, we have a `Parameter.HARDWARE_MODULATION` that we use in other instruments as well.

Let's verify this with measurements team.